core/src/main/java/io/grpc/internal/PickFirstLeafLoadBalancer.java

```diff
@@ -610,27 +612,26 @@ public PickResult pickSubchannel(PickSubchannelArgs args) {
   }
 
   /**
-   * Index as in 'i', the pointer to an entry. Not a "search index."
+   * This contains both an ordered list of addresses and a pointer(i.e. index) to the current entry
```

Comment:
End in a period, otherwise it merges with the next line.

Reply:
Done